src/Automation/ScanResultsAssembler.cs

```diff
@@ -11,14 +12,14 @@ namespace Axe.Windows.Automation
     /// <summary>
     /// Provides methods used to assemble a <see cref="ScanResults"/> object
     /// </summary>
-    public static class ScanResultsAssembler
+    public class ScanResultsAssembler : IScanResultsAssembler
```

Comment:
This class should be internal as well because it is not intended to be used by clients.

Reply:
Now internal